web-client/src/views/CreateOrder/TextEditor.tsx

```diff
@@ -82,6 +82,12 @@ export const TextEditor = ({
             const fullDelta = editor.getContents();
             const documentContents = editor.getText();
             const converter = new QuillDeltaToHtmlConverter(fullDelta.ops, {
+              // We add custom CSS styles to maintain tabbing across save and reload.
+              // (Note that preserveWhitespace does not work: https://github.com/ustaxcourt/ef-cms/pull/1408.)
+              // TODO: preserve toolbar-generated indentation across save and reload.
```

Comment:
I was hoping that maintaining the indentation level--which the user can define by using the toolbar--would be easy enough to restore upon edit. Unfortunately, it isn't, and I don't want to spend too much time on it since this is already an improvement.